hip-0000-template.md

```diff
@@ -1,72 +1,491 @@
+The proposed HIP introduces a partitioning feature to effectively and efficiently manage subsets of multiple fungible tokens within a single token ID, meeting business/compliance requirements and reducing overhead costs. 
+
+At present, when referring to fungible supply: 
+
+* Fungible tokens are identical to one another, making it impossible to distinguish tokens within the supply. This limits their use to situations where uniformity and interchangeability are more important than individual token identity.
+* The execution of native primitives[^2] on fungible tokens operates on the entire supply in a uniform manner. As such, any action or operation undertaken on these tokens will impact all other tokens in the supply equally, further emphasizing the indistinguishable nature of fungible tokens.
+
+When referring to non-fungible tokens (NFT) supply: 
+
+* Each NFT has unique properties and metadata. An NFT represents a unique asset or item and cannot be interchanged with other NFTs on the market. 
+* Execution of native primitives<sup>2</sup> affects the single non-fungible tokens. The inability to perform bulk execution of native primitives on multiple NFTs limits scalability due to the computational cost of performing individual operations on a large number of NFTs.
+
+When referring to non-fungible tokens (NFT) collections: 
+
+* Collections of NFTs offer a way to group together multiple unique assets or items,  but with a supply limited to 1.
+
+The partitioning feature would align with regulatory and compliance requirements of securities and derivatives, making it possible to attract more use cases and developers from “traditional finance” to the platform. Wallets and exchanges would display the partitioned fungible tokens as a single, unified entity, similar to how ERC-1155 collections are represented today. 
+
+The partitioned fungible tokens would contain various types of tokens with different properties, characteristics, and use cases, all bundled together under a common identifier. 
+
+This approach enables a seamless user experience and simplifies the management of diverse token types within a token ID, providing users with a familiar and consistent user interface similar to ERC-1155.
+
+Overall, introducing the ability to partition fungible tokens would enable the Hedera network to better cater to multiple industries and would provide: 
+
+
+
+1. **Flexibility** \
+This would allow greater flexibility with potential access to new use cases and markets, allowing Hedera to meet the diverse needs of different industries and additional applications that the existing specifications and services for fungible and non-fungible tokens don't currently meet.
+2. **Regulatory Compliance** \
+Traditional financial institutions and enterprises often require a high level of regulatory compliance to ensure security, transparency, and accuracy that existing specifications and services do not address. 
```

Comment:
Can you share an example of how partitioning tokens would facilitate easier regulatory compliance?

Reply:
The `move` primitive enables moving tokens across partitions, effectively changing their metadata/grouping without adjusting the supply or emitting a mint or burn event.